packages/beacon-node/test/perf/network/noise/sendData.test.ts

```diff
@@ -24,15 +26,18 @@ describe("network / noise / sendData", () => {
     bench({
       id: `send data - ${numberOfMessages} ${messageLength}B messages`,
       beforeEach: async () => {
-        const peerA = await createSecp256k1PeerId();
-        const peerB = await createSecp256k1PeerId();
-        const noiseA = noise()({logger: defaultLogger()});
-        const noiseB = noise()({logger: defaultLogger()});
+        const privateKeyA = await generateKeyPair("secp256k1");
+        const privateKeyB = await generateKeyPair("secp256k1");
+        const peerA = peerIdFromPrivateKey(privateKeyA);
+        const peerB = peerIdFromPrivateKey(privateKeyB);
+        const upgrader = {getStreamMuxers: () => new Map()} as Upgrader;
```

Comment:
this is the minimal interface required by noise `Upgrader` based on https://github.com/ChainSafe/js-libp2p-noise/pull/482